@here/harp-mapview/lib/Tile.ts

```diff
@@ -256,6 +257,46 @@ interface PolygonFadingParameters extends FadingParameters {
     lineFadeFar?: number;
 }
 
+/**
+ * Proxies a Tile with a given offset (a numeric value to represent the longitudinal shift of the
+ * tile (a multiple of 360 degrees)). The given offset should be non zero, because we don't need to
+ * proxy the Tile at offset 0 (a Tile is by definition at offset 0).
+ *
```

Comment:
Please explain the uses case in a bit more detail, since it messes up the code quite a bit: 
* Why was Tile.tile added
* Why is the proxy needed
* Why do we use cloned objects
* Why are cloned TextElements/objects acceptable (not many objects at that level)

Reply:
> Why do we use cloned objects

I don't thnk that 3JS can render one object in two places in scene.
Otherwise, we would need several render passes - each one for each "globe" when each globe has different x-offset.

On the other hand, we may reuse underlying geometry & materials so these clones should be smart.

Reply:
- Tile.tile is described below, see get tile(), but can add here if you think that is necessary.
- Will update the comment to otherwise fix what you mentioned (include what @zbigg mentioned)

Reply:
I added these comments to the TileProxy class, see: https://github.com/heremaps/harp.gl/pull/317/commits/e9d3bc08a9761f80566d8a26d101f672f7f4ae34#diff-bec3ef09a66148da8cb27c7acb7625b7
So marking as resolved.